inginious/frontend/plugins/manual_scoring/static/js/code_area.js

```diff
@@ -11,29 +11,41 @@ const LANGUAGES = {
     "rst": "rst"
 };
 
+const environmentTypes = {
+    NOTEBOOK: "notebook_file",
+    FILE_MULTI_LANG: "code_file_multiple_languages",
+    MULTI_LANG: "code_multiple_languages"
+}
+
 class CodeArea {
 
-    constructor(multiLangCodeDivId, notebookDivId = null, environmentType = "") {
-        this.environmentType = environmentType
-        this.multilangCodeArea = $(`#${multiLangCodeDivId}`);
+    constructor(environmentType) {
+        this.environmentType = environmentType;
+        this.multilangCodeArea = $(`#${CODE_AREA_ID}`);
+        this.notebookDiv = $(`#${NOTEBOOK_CODE_AREA_ID}`);
+        this.fileDownloadArea = $(`#${FILE_MULTI_LANG_ID}`);
         this.codeMirror = null;
-        if (notebookDivId)
-            this.notebookDiv = $(`#${notebookDivId}`);
     }
 
     displayCodeArea() {
-        if (this.isNotebook()) {
-            this.getNotebookCodeDataAndRender();
-        } else {
-            this.showMultiLangCodeArea();
-            this.disabledCodeMirrorEdit();
+        switch (this.environmentType) {
+            case environmentTypes.NOTEBOOK:
+                this.getNotebookCodeDataAndRender();
```

Comment:
There is something wrong with  the Notebook renderer. It does not render the notebook correctly. Probably some CSS styles were changed that affect the way it is render.
![image](https://user-images.githubusercontent.com/22863695/117215907-903c5380-adc4-11eb-9244-d45a278c57ee.png)